packages/coreutils/src/path.ts

```diff
@@ -69,13 +69,15 @@ export namespace PathExt {
    * @returns the last part of the path, sans extension.
    */
   export function stem(path: string): string {
+    // TODO: This will always fail?
+    // TODO: Exclamation marks are only here to allow it to compile
```

Comment:
Does this function work at all? It was completely untested/unused, and seems likely to fail?

Reply:
I'd say remove it.

Reply:
Removed.